NuGet.Config

```diff
@@ -1,11 +1,6 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <clear />
```

Comment:
> <clear /> [](http://example.com/codeflow?start=3&length=10)

the <clear/> is needed for security reasons.

NuGet.Config - Missing <clear/> statement.

Reply:
Yes, another good catch, another merge problem. :)